core/src/components/datetime/test/showDaysOutsideCurrentMonth/index.html

```diff
@@ -0,0 +1,345 @@
+        --highlight-border-radius: 16px;
+        --fade-background-rgb: 245, 235, 247;
+
+        background-color: rgb(245, 235, 247);
+      }
+      .custom-grid-wheel::part(wheel-item),
+      ion-picker-column-option {
+        color: rgb(255, 134, 154);
+      }
+
+      .custom-grid-wheel::part(wheel-item active),
+      ion-picker-column-option.option-active {
+        color: rgb(128, 30, 171);
+      }
+
+      /*
+        * Custom Datetime Day Parts
+        * -------------------------------------------
+        */
+
+      #custom-calendar-days::part(calendar-day) {
+        background-color: #ffe2e6;
+
+        color: #da5296;
+
+        margin: 2px 0;
+      }
+
+      #custom-calendar-days::part(calendar-day today) {
+        color: #8462d1;
+      }
+
+      #custom-calendar-days::part(calendar-day):not(.calendar-day-hidden-day):focus {
+        background-color: rgb(154 209 98 / 0.2);
+        box-shadow: 0px 0px 0px 4px rgb(154 209 98 / 0.2);
+      }
+
+      #custom-calendar-days::part(calendar-day disabled) {
+        background: rgba(0 0 0 / 0.2);
+        color: black;
+      }
+
+      /*
+        * Custom Material Design Datetime Day Parts
+        * -------------------------------------------
+        */
+
+      #custom-calendar-days.md::part(calendar-day active),
+      #custom-calendar-days.md::part(calendar-day active):focus {
+        background-color: #9ad162;
+        border-color: #9ad162;
+        color: #fff;
+      }
+
+      #custom-calendar-days.md::part(calendar-day today) {
+        border-color: #8462d1;
+      }
+
+      /*
+        * Custom iOS Datetime Day Parts
+        * -------------------------------------------
+        */
+
+      #custom-calendar-days.ios::part(calendar-day active),
+      #custom-calendar-days.ios::part(calendar-day active):focus {
+        background-color: rgb(154 209 98 / 0.2);
+        color: #9ad162;
+      }
+
+      body {
+        padding: 20px;
+      }
+
+      ion-datetime {
+        border: 1px solid black;
+      }
+    </style>
+  </head>
+
+  <body>
+    <ion-app>
+      <ion-header translucent="true">
+        <ion-toolbar>
+          <ion-title>Datetime - ShowDaysOutsideCustomMonth</ion-title>
+        </ion-toolbar>
+      </ion-header>
+      <ion-content class="ion-padding">
+        <div class="grid">
+          <div class="grid-item">
+            <h2>Grid Style</h2>
+            <ion-datetime id="custom-grid" value="2020-03-14T14:23:00.000Z" class="custom-grid-wheel"></ion-datetime>
+          </div>
+
+          <div class="grid-item">
+            <h2>Grid Style</h2>
+            <ion-datetime id="custom-calendar-days" value="2023-06-15" presentation="date"></ion-datetime>
+          </div>
+        </div>
+
+        <div class="grid">
+          <div class="grid-item">
+            <h2>Disable Specific Date</h2>
+            <ion-datetime id="specificDate" value="2021-10-01" showdaysoutsidecurrentmonth="true"></ion-datetime>
```

Comment:
```suggestion
            <ion-datetime id="specificDate" value="2021-10-01" show-adjacent-days="true"></ion-datetime>
```

We should use kebab case here